src/Domain/CoordinateMaps/Composition.cpp

```diff
@@ -299,6 +366,190 @@ Composition<Frames, Dim, std::index_sequence<Is...>>::inv_jacobian_impl(
       inv_jacobians);
 }
 
+#ifdef SPECTRE_AUTODIFF
+// potential optimization: merge with inv_jacobian_impl
+template <typename Frames, size_t Dim, size_t... Is>
+template <typename DataType>
+InverseHessian<DataType, Dim, tmpl::front<Frames>, tmpl::back<Frames>>
+Composition<Frames, Dim, std::index_sequence<Is...>>::inv_hessian_impl(
```

Comment:
Can you add the latex math for composing Hessians somewhere? Maybe to the general class documentation of the `CoordinateMap` class and then just point to that from elsewhere?

Should we just have a function that computes coords, jacobian, coord velocity (only moving maps), and Hessian. From those we can then compute the inverse Jacobian by numerical inversion and the inverse Hessian from the Hessian and inverse Jacobian with some identities. I think that would be both more efficient and simpler to understand. If we want, we could have the inverse Hessian from the inverse Jacobian and Hessian as a separate function since I think we mostly need the Hessian. What do you think?